gobblin-data-management/src/main/java/org/apache/gobblin/data/management/trash/TrashFactory.java

```diff
@@ -59,19 +59,12 @@ public static Trash createTrash(FileSystem fs, Properties props) throws IOExcept
    */
   public static Trash createTrash(FileSystem fs, Properties props, String user)
       throws IOException {
-    if(props.containsKey(TRASH_TEST) && Boolean.parseBoolean(props.getProperty(TRASH_TEST))) {
-      LOG.info("Creating a test trash. Nothing will actually be deleted.");
-      return new TestTrash(fs, props, user);
-    }
-    if(props.containsKey(SIMULATE) && Boolean.parseBoolean(props.getProperty(SIMULATE))) {
-      LOG.info("Creating a simulate trash. Nothing will actually be deleted.");
-      return new MockTrash(fs, props, user);
-    }
-    if(props.containsKey(SKIP_TRASH) && Boolean.parseBoolean(props.getProperty(SKIP_TRASH))) {
```

Comment:
Looks like we no longer handle SKIP_TRASH and SIMULATE, and replace them with custom trash classes. Is there any effect on existing jobs/backward compatibility?

Reply:
Can you please reframe your question? Why do we no longer handle SKIP_TRASH and SIMULATE? I moved this piece of code to createTestMockOrImmediateDeletionTrash()

Reply:
oh, I see. it was already checking it there.